src/test/suite/didactUriCompletionItemProvider.test.ts

```diff
@@ -79,6 +79,7 @@ suite("Didact URI completion provider tests", function () {
 			listOfCompletions.forEach(function(stringToTest: string){
 				afterEach( async () => {
 					await removeFilesAndFolders(testWorkspace, foldersAndFilesToRemove);
+					delay(250);
```

Comment:
wouldn't it be better to remove the async/await instead of adding a delay? did not test yet but seems weird to be to try being asynchronous but adding a delay because we actually aren't

Reply:
Maybe. I was just throwing anything at this problem to see if I could get it fixed.

Reply:
The removeFilesAndFolders function relies on the workspace.fs instance to recursively delete files and folders (i.e. `await vscode.workspace.fs.delete(delUri, {recursive:true});`) I have no clue how to make that non async/await

Reply:
Adding the delay doesn't really change anything (we use the function elsewhere in this set of tests and it works fine).